src/browser/atom-application.coffee

```diff
@@ -9,7 +9,7 @@ app = require 'app'
 dialog = require 'dialog'
 shell = require 'shell'
 fs = require 'fs-plus'
-ipc = require 'ipc'
+ipc = require 'ipc-main'
```

Comment:
I *think* this file uses `ipc-main`, but if someone could double-check, that would be great.